source/data/tier1data/features/collaboration.rst

```diff
@@ -0,0 +1,192 @@
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Action
+     - Read
+     - Write
+     - Own
+   * - View
+     - ✓
+     - ✓
+     - ✓
+   * - Download (data object)
+     - ✓
+     - ✓
+     - ✓
+   * - Copy
+     - ✓
+     - ✓
+     - ✓
+   * - Edit/overwrite (data object)
+     -
+     - ✓
+     - ✓
+   * - Create new files/collections (collection)
+     -
+     - ✓
+     - ✓
+   * - Metadata
+     - View
+     - Edit
+     - Edit
+   * - Rename
+     - 
+     - 
+     - ✓
+   * - Move
+     - 
+     - 
+     - ✓
+   * - Delete
+     - 
+     - 
+     - ✓
+   * - Change permissions
+     -
+     -
+     - ✓
+
+
+'Null' access allows no actions. Assigning 'null' permissions is equivalent to removing existing permissions. 
+    
+Each permission relates to a specific collection or data object.  
+For example, you can give a group access to a specific collection, but not to the subcollections or data objects under it.
+
+- Chemistry (Mary - read)
+    - ExperimentA (Mary - write)
+        - result1.txt 
+        - result2.txt 
+        - result3.txt
+    - ExperimentB
+        - result1.txt 
+        - result2.txt
+        - result3.txt
+
+In this example, Mary will see the collection Chemistry and its subcollection ExperimentA, but not ExperimentB.
+Furthermore, she will not be able to see the data objects inside of ExperimentA.
+However, she can upload new files to this collection herself. 
+
+It's important to stress that one object can have multiple permissions:
+
+- CollectionA
+    - GroupA: read access
+    - GroupB: read access
+    - GroupC: write access
+    - GroupD: own access
+
+This way, it is possible that a person can derive access to an object from multiple permissions.
+If Mary is part of both GroupA and GroupC, she will have write access to CollectionA (because this is the highest level of access given to her).  
+
+
+Inheritance and recursiveness
+=============================
+
+If we would have to manage permissions individually for every new collection or data object, this would take a long time.
+To solve this problem, we can make use of inheritance and recursiveness.
+
+**Inheritance** is an attribute of a collection. If it is turned on, all collections or data objects that get created/uploaded under that collection inherit its permissions automatically:
+
+- Chemistry
+    - ExperimentA (Chemistry_data_providers - own, inheritance - on)
+        - Newfile.txt (Chemistry_data_providers - own)
+        - Newcollection (Chemistry_data_providers - own, inheritance - on)
+
+If inheritance is turned off, permissions from the parent collection are not applied.
+The person who creates/uploads new data objects/collections gets own access by default, but no other permissions are added:
+
+- Chemistry
+    - ExperimentA (Chemistry_data_providers - own, inheritance - off)
+        - Newfile.txt (John - own)
+        - Newcollection (John - own)
```

Comment:
Fix formatting:
```suggestion
- Chemistry
  - ExperimentA (Chemistry_data_providers - own, inheritance - off)
    - Newfile.txt (John - own)
    - Newcollection (John - own)
```

Reply:
See above: I can't seem to create a nested unordered list except by using hyphens and 4 spaces

Reply:
I forgot to add blank lines between the indented blocks. Sorry for the confusion. I pushed the fix to your branch in https://github.com/hpcleuven/VscDocumentation/pull/349/commits/adb42f02ea257dbf03ff030dcc41c1446147c8da